bin/ignite

```diff
@@ -1,9 +1,31 @@
 #!/usr/bin/env node
 
-const sourceDir = `${__dirname}/../src/cli`
+/* tslint:disable */
```

Comment:
who's a bad boy?  😈

Reply:
I have to do this here because it has to be ES5 compatible (for reasons).